examples/CMakeLists.txt

```diff
@@ -13,3 +13,19 @@ target_include_directories(fortran_hello_world PRIVATE ${MPI_Fortran_INCLUDE_DIR
 set_target_properties( fortran_hello_world PROPERTIES INSTALL_RPATH "../lib")
 
 install( TARGETS fortran_hello_world )
+
+
+
+#
+# Simple Trixi Controller
+#
+add_executable(simple_trixi_controller simple_trixi_controller.c)
+
+target_link_libraries(simple_trixi_controller PRIVATE ${PROJECT_NAME})
```

Comment:
I don't understand - how is `${PROJECT_NAME}` set? Or this this CMake file still just included by the main CMake file?

Reply:
Ah, yes, I just checked, it is :-) NVM!